recipes/srm/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "1.2.15" %}
+
+package:
+  name: srm
+  version: {{ version }}
+
+source:
+  url: http://download.sourceforge.net/srm/srm-{{ version }}.tar.gz
+  sha256: 7583c1120e911e292f22b4a1d949b32c23518038afd966d527dae87c61565283
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
```

Comment:
Does this need make?

Reply:
Hmm...we've been using the system copy for a while. Are we changing that?

Reply:
I think so, especially at the lower level. CC @mariusvniekerk who was doing some work on these things yesterday.

Reply:
I think we don't have a consistent policy on this, so it has pretty much been up to the library author

Reply:
In that case I'm inclined to leave it out unless people feel strongly about it.

Reply:
OK!